holidays/holiday_base.py

```diff
@@ -786,7 +786,12 @@ def _init_translation(self) -> None:
         supported_languages = set(self.supported_languages)
         if self._entity_code is not None:
             fallback = self.language not in supported_languages
-            languages = [self.language] if self.language in supported_languages else None
+            if not fallback and self.language is not None:
```

Comment:
I'd like to have a comment explaining why we have those if/else here.

Reply:
AFAIK this was done so that child entity can have different `default_language` from its parent entity (in this case, `AX` got to keep `sv_FI` instead of `fi` uses in Finland)

Lemme see if I can make a mobile edit for this real quick